packages/server/lib/hooks/connection/post-connection.ts

```diff
@@ -74,8 +75,20 @@ async function execute(createdConnection: RecentlyCreatedConnection, provider: s
         const handler = handlers[`${provider.replace(/-/g, '')}PostConnection`];
 
         if (handler) {
+            logCtx = await logContextGetter.create(
+                { operation: { type: 'auth', action: 'post_connection' }, message: 'Start internal post connection' },
```

Comment:
See note above about adding "script.

"external" vs "internal" might be confusing to the user. Maybe for both it is just "Start post connection script". I think originally we didn't log this (which was intentional) and maybe since it is internal we should log it and on failure we just keep an internal `logger` log so we can know about it. A user might be confused to see this log that they wouldn't necessarily know anything about.

Reply:
My own feeling about this is that it's part of the system that interact with the third parties so it should be logged. And it's not even enough imo since we don't handle the buffered logs in the proxy call.

> A user might be confused to see this log that they wouldn't necessarily know anything about.

I feel like it's even worse if we only show errors to the users, they could think it never works

Also, no logs can mean "nothing was executed" or "something was a success "or "something crashed", which is not super nice for observability. And stdout don't scale indefinitely either. 

---

Happy to involve @bastienbeurier on this. The context, we currently log: 
- external post connection script: failure and success  
- internal post connection: failure

**We don't we log successful internal post connection, should we do it or keep it as is?**
(side note: we never log the absence of post connection)

Alternative, should this log be part of the auth:create_connection log instead?

Reply:
IMO, we should err on the side of transparency, logging everything but being very explicit about what is executed by the platform script vs. what is executed by a user script.